.github/workflows/arm64.yml

```diff
@@ -26,7 +26,6 @@ env:
   GO_VERSION: "1.24.7"
   TESTSTAT_VERSION: v0.1.25
   DESTDIR: ./build
-  SETUP_BUILDX_VERSION: edge
   SETUP_BUILDKIT_IMAGE: moby/buildkit:latest
```

Comment:
Perhaps downgrade this one as well?

Reply:
Ah, thanks @thaJeztah . I just saw your message. For context, the last good run I had was https://github.com/moby/moby/actions/runs/18142797653/job/51637569553

xrefing env changes from there.

Reply:
Oh, I was missing something simple. `14d1ccb` is the shortsha for buildkit v0.25.0.